Makefile

```diff
@@ -5,7 +5,7 @@ default:
 .PHONY: default
 
 install:
-	ln -s $(abspath bin/kubectl-crossplane-stack-*) $(INSTALL_DIR)/
+	ln -sf $(abspath bin/kubectl-crossplane-stack-*) $(INSTALL_DIR)/
```

Comment:
Since we are adding a `-f` for the main init, I would probably prefer `-i` here instead of `-f`, so that it asks for confirmation to clobber